### DISCUSSION TITLE
Import domain Certificate

### DISCUSSION THREAD

**Discussion by kvkamin** [Security-Onion-Solutions/securityonion#11061] [2.4] [answered]
I would like to use a wildcard cert we use for our domain.  I don't see any documentation on the process. Also, is nginx the only place a cert is used?

Karl

**Comment by kvkamin** [comment]
a little research shows that /etc/pki/managerssl.crt is same as /etc/pki/nginx/server.crt in so-nginx container. And /opt/so/conf/nginx/nginx.conf references the same cert and key.  Is it simply a matter of replacing new cert and key in /etc/pki/managerssl.key,crt ? and waiting for salt to replace them in the so-nginx container?

**Reply by TOoSmOotH** [reply, marked as answer]
Under Configuration go into advanced mode and under nginx you will see ssl. You can paste the contents of the certs there.

**Reply by kvkamin** [reply]
Thanks!  I didn't see advanced under options on top!  Fixed.

K

**Reply by CharlieMK** [reply]
That works for manual Config, but in 2.3 I replaced the cert using a script. Is that still supported or will that mess up the Config?

**Reply by dougburks** [reply]
@CharlieMK Please start a new discussion rather than replying to somebody else's discussion.
